lib/internal/modules/esm/loader.js

```diff
@@ -187,10 +182,7 @@ class ModuleLoader {
     }
   }
 
-  async eval(
-    source,
-    url = pathToFileURL(`${process.cwd()}/[eval${++this.evalIndex}]`).href,
-  ) {
+  async eval(source, url, isEntryPoint = false) {
```

Comment:
Not sure why this method lacks a JSDoc, do you mind adding one?

Reply:
Hmm, didn't see this, feel free to follow up and add a comment if you like.

Reply:
Weird! I just looked through the history, and it seems this method never got a jsdoc when pretty much everything else did.